packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -257,7 +258,7 @@ export class GadgetConnection {
       fetch: this.fetch,
       exchanges: [
         dedupExchange,
-        fetchExchange,
+        multipartFetchExchange,
```

Comment:
Any performance issues to using this over `fetchExchange`? If yes, perhaps down the road we could provide a set of default exchanges from the generated API clients, based on whether or not there are file fields.

Reply:
The multipart fetch exchange does exactly the same thing as the fetchExchange when there's no files involved, it only does multipart stuff if there are files, so I don't think so! Woop woop!